webhook/command_response.go

```diff
@@ -0,0 +1,54 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/micromdm/micromdm/connect"
+	"github.com/micromdm/micromdm/pubsub"
+	"github.com/pkg/errors"
+)
+
+const contentType = "application/x-apple-aspen-mdm"
+
+type CommandResponse struct {
```

Comment:
It would be nice to keep this struct type a bit more generic so that other subscriptions can be added and forwarded (for example Checkin events like TokenUpdate and Checkout)

Reply:
I'm making assumptions about the event type and how to unmarshal it, so I'm not sure how to cleanly generalize in this case.